ci/gpu/build.sh

```diff
@@ -37,6 +43,8 @@ conda list
 # BUILD - Build libcuml and cuML from source
 ################################################################################
 
+git submodule update --init --recursive
```

Comment:
Why is this necessary? The jenkins checkout process should handle this.

Reply:
While working on building docs on Friday, Jenkins would for some reason get the wrong version of the submodule causing the build to fail. I can remove and see if it happens on this build too, I just copied what worked.

Reply:
Build failed the same way on Friday without it.